### DISCUSSION TITLE
I get this error building anchor project: package `solana-program v1.16.14` cannot be built because it requires rustc 1.68.0 or newer, while the currently active rustc version is 1.62.0-dev

### DISCUSSION THREAD

**Discussion by jsoneaday** [otter-sec/anchor#2638] [Q&A] [answered]
I am getting this error despite being on rustc 1.72.1. How can I force my project to use 1.68?

**Comment by jsoneaday** [comment]
I tried running rustup override set 1.68.0 but same error. When I check the rustc version I see 1.68.0.

```
(base) davidchoi@Davids-MacBook-Pro-2 puppet % rustc --version
rustc 1.68.0 (2c8cc3432 2023-03-06)
```

**Reply by buffalojoec** [reply, marked as answer]
Hello, your Solana CLI version is likely a bit older than 1.16, so it likely has an older version of Rust. 

Solana CLI attempts to override the active `rustc` version when you run one of the build commands (ie. `cargo build-sbf`). 

Try upgrading your CLI

**Reply by jsoneaday** [reply]
That was it. But what version of yarn are you guys running? I tried doing a yarn install on root of workspace but still not installing deps. I am on yarn 3.4.1

**Reply by buffalojoec** [reply]
I'm not sure, but yarn won't bring the Solana CLI down. I think that might be a separate question for Q & A!
